api/pom.xml

```diff
@@ -39,9 +39,10 @@
 		<testResources>
 			<testResource>
 				<directory>src/test/resources</directory>
-				<filtering>true</filtering>
+				<filtering>false</filtering>
```

Comment:
What does this do?

Reply:
We added some test zip files into the test directory, and Maven was doing variable-replacement filtering on these files, which was corrupting them.